loadtest/run.go

```diff
@@ -30,7 +30,7 @@ import (
 func RunTest(test *TestRun) error {
 	r := rand.New(rand.NewSource(0))
 
-	interruptChannel := make(chan os.Signal)
+	interruptChannel := make(chan os.Signal, 1)
```

Comment:
Can you explain why this change is here?

Reply:
This is unrelated to the PR. I ran into a warning that `signal.Notify()` could miss some signals if the channel was not buffered. I read the relevant part in the [godoc](https://golang.org/pkg/os/signal/#Notify) and it was suggested to use a buffered channel.

Reply:
Makes sense! Learned something new :)